core/src/main/java/hudson/init/InitMilestone.java

```diff
@@ -82,13 +85,29 @@
      * By this milestone, all programmatically constructed extension point implementations
      * should be added.
      */
-    EXTENSIONS_AUGMENTED("Augmented all extensions"), // TODO nothing attains() this so when does it actually happen?
+    EXTENSIONS_AUGMENTED("Augmented all extensions"),
+
+    /**
+     * By this milestone, all the configurations are loaded from file system
```

Comment:
"... all the *system* configuration ...", to make sure this does not involve jobs

Reply:
```suggestion
     * By this milestone, all the system configurations are loaded from file system
```